src/yamlfix/entrypoints/cli.py

```diff
@@ -18,21 +18,62 @@ def _format_file_list(files: Tuple[TextIOWrapper]) -> str:
     return "\n  - ".join([""] + file_names)
 
 
-@click.command()
-@click.version_option(version="", message=version.version_info())
-@click.option("--verbose", is_flag=True, help="Enable verbose logging.")
+@click.command()  # type: ignore
+@click.version_option(version="", message=version.version_info())  # type: ignore
+@click.option("--verbose", is_flag=True, help="Enable verbose logging.")  # type: ignore
 @click.option(
     "--check",
     is_flag=True,
     help="Check if file(s) needs fixing. No files will be written in this case.",
-)
-@click.argument("files", type=click.File("r+"), required=True, nargs=-1)
-def cli(files: Tuple[str], verbose: bool, check: bool) -> None:
+)  # type: ignore
+@click.option(
+    "--none-representation",
+    is_flag=False,
+    type=str,
+    help="Represent empty / none values with the provided string.",
+)  # type: ignore
+@click.option(
+    "--quote-representation",
+    is_flag=False,
+    type=str,
+    help="Use the provided character as the default quotation mark.",
+)  # type: ignore
+@click.option(
+    "--quote-everything", is_flag=True, help="Quote all keys and simple string values."
+)  # type: ignore
+@click.option(
```

Comment:
So far I don't understand very well the difference between `--quote-everything` and `--quote-basic-values`. I'm sure that with the tests it will be clearer. Just a note to keep in mind when writing the docs of these features